website/docs/gator.md

```diff
@@ -0,0 +1,180 @@
+---
+id: gator
+title: The gator CLI
+---
+
+The `gator` CLI is a tool for testing Gatekeeper ConstraintTemplates and
+Constraints on Kubernetes objects.
+
+We plan on adding more subcommands in the future.
+
+## Installation
+
+To install `gator`, you may either
+[download the binary](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.7.0)
+relevant to your system or build it directly from source.
+
+To build from source:
+```
+go get github.com/open-policy-agent/gatekeeper/cmd/gator
+```
+
+## The `gator test` subcommand
+
+### Writing Test Suites
+
+`gator test` organizes tests into three levels: Suites, Tests, and Cases:
+
+- A Suite is a file which defines Tests.
+- A Test declares a ConstraintTemplate, a Constraint, and Cases to test the
+  Constraint.
+- A Case defines an object to validate and whether the object is expected to
+  pass validation.
+
+Any file paths declared in a Suite are assumed to be relative to the Suite
+itself. Absolute paths are not allowed. Thus, it is possible to move around a
+directory containing a Suite, and the files it uses for tests.
+
+### Suites
+
+[An example Suite file](https://github.com/open-policy-agent/gatekeeper-library/blob/8765ec11c12a523688ed77485c7a458df84266d6/library/general/allowedrepos/suite.yaml).
```

Comment:
Do we want to pin it to a commit or `master` so we don't have to update this doc if there are changes in that file?

Reply:
This is pinned to a commit. GitHub URLs that include `/blob/` are permalinks to a file at a commit. The long string of letters/numbers is the commit hash.

Reply:
I think Rita is suggesting we pin to a branch, as opposed to a specific commit.

The idea being that if we were to update `library/general/allowedrepos/suite.yaml` in the future, the user would be taken to the updated version when clicking the link from the docs.

Reply:
I like having it pinned to a specific commit as it means if we move the file, the link doesn't just break